service/src/main/resources/static/regions.yml

```diff
@@ -102,19 +105,4 @@ regions:
       - gcp.asia-south1
       - gcp.asia-northeast3
       - gcp.asia-southeast1
-      - gcp.australia-southeast1
-      - gcp.asia-east1
-      - azure.eastasia
-      - azure.southeastasia
-      - azure.centralindia
-      - azure.southindia
-      - azure.koreacentral
-      - azure.australiaeast
-  - name: specific-datacenter
-    description: Container for each individual datacenter
-    regions:
-    # TODO: Add all the others here.
```

Comment:
OK, I think I understand. You want to define a "region" with the name of the datacenter to represent it in the region graph, so you have a homogeneous graph.

I wonder if it would be cleaner to model the region graph as only regions and not separate out the data centers. You could distinguish by prefixing the non-datacenter ones with "geo." or some such. Then you wouldn't need the datacenters.yml or to keep it in sync with the region file.

Reply:
Seeing how this is shaping up, I think that makes sense. Do we want to make that change as part of this review or can it be a follow-up item?